### PR TITLE
layers: Fix syncval access mask tables for transform feedback

### PR DESCRIPTION
**Pull request by jeremyg-lunarg**
Note: the scope of this change has expanded...

    layers: Sync syncval snippet tables with spec v1.2.169
    
    This pulls in changes to ray tracing pipeline stages and access
    masks.
    
    VK_ACCESS_TRANSFORM_FEEDBACK_COUNTER_READ_BIT_EXT lost an allowed
    pipeline stage in commit 5f585ae24. Note this is in a pending
    change to the spec that is not yet in v1.2.169
    
    A growing amount of manual fixups is required to pull this stuff
    in to the script. I've noted them, but we really need to find
    a better way to get these tables.
    
    This fixes github issue #2542

### CODE REVIEW

**Comment by ShabbyX** [comment]
Thanks for the quick fix!

File: scripts/sync_val_gen.py [hunk 1]
```diff
@@ -98,7 +98,7 @@
 ifdef::VK_EXT_transform_feedback[]
 |ename:VK_ACCESS_TRANSFORM_FEEDBACK_WRITE_BIT_EXT             | ename:VK_PIPELINE_STAGE_TRANSFORM_FEEDBACK_BIT_EXT
 |ename:VK_ACCESS_TRANSFORM_FEEDBACK_COUNTER_WRITE_BIT_EXT     | ename:VK_PIPELINE_STAGE_TRANSFORM_FEEDBACK_BIT_EXT
-|ename:VK_ACCESS_TRANSFORM_FEEDBACK_COUNTER_READ_BIT_EXT      | ename:VK_PIPELINE_STAGE_DRAW_INDIRECT_BIT
+|ename:VK_ACCESS_TRANSFORM_FEEDBACK_COUNTER_READ_BIT_EXT      | ename:VK_PIPELINE_STAGE_DRAW_INDIRECT_BIT, or ename:VK_PIPELINE_STAGE_TRANSFORM_FEEDBACK_BIT_EXT
```
Comment (jzulauf-lunarg) [inline scripts/sync_val_gen.py:101]: Should update the comment `Snipped from chapters/synchronization.txt -- tag v1.2.135` to reflect the spec version of the excerpt.

**Review by jzulauf-lunarg — CHANGES_REQUESTED** [review]
I understand the desire for a quick turn, but are all of the snippets updated and from the same tag?  I can live with just correctly attributed locally, but we shouldn't let the excerpts become inconsistent.

File: scripts/sync_val_gen.py (hunk 1 above)
Comment (jeremyg-lunarg) [reply scripts/sync_val_gen.py:101]: Whelp, that'll make this more difficult. https://gitlab.khronos.org/vulkan/vulkan/-/commit/512cddc4042a0d280ce39286d67190790b4888b9  has not merged to vulkan/vulkan main yet so there is not yet a released header version that includes it.

Additionally, there's been changes to the snippet sections at some point that will either require code changes or manual fixups after pasting in the new text.

Comment (jzulauf-lunarg) [reply scripts/sync_val_gen.py:101]: Then just update the tag line with the comment that it's effectively at a tag with cherry-picked fix or something.  

Especially if there's another change that will clean this up on-deck ;)

Comment (jeremyg-lunarg) [reply scripts/sync_val_gen.py:101]: It is actually proving helpful to update for this change separately before that next thing... We REALLY need to get this info in a more machine readable format, like the VUIDS.

Comment (jzulauf-lunarg) [reply scripts/sync_val_gen.py:112]: Didn't mean for this to blow up on you like this... but looks like it was needed.

**Review by jzulauf-lunarg — APPROVED** [review]
LGTM.

**Comment by ncesario-lunarg** [comment]
LGTM!

**Pull request merged**